src/main/resources/messages/RsBundle.properties

```diff
@@ -108,6 +190,27 @@ settings.rust.inlay.hints.title.method.chains=Method chains
 settings.rust.inlay.hints.title.types=Types
 settings.rust.inlay.parameter.hints.only.smart=Only smart hints
 
+settings.rust.rustfmt.builtin.formatter.label=Use rustfmt instead of built-in formatter
+settings.rust.rustfmt.name=Rustfmt
+settings.rust.rustfmt.run.on.save.label=Run rustfmt on Save
+
+settings.rust.toolchain.download.rustup.dialog.title=Downloading Rust Standard Library
+settings.rust.toolchain.download.rustup.link=Download via Rustup
+settings.rust.toolchain.download.rustup.progress.text=Installing using Rustup...
+settings.rust.toolchain.expand.macros.comment=Allow plugin to process macro invocations to extract information for name resolution and type inference
+settings.rust.toolchain.expand.macros.disable.label=Disable (select only if you have problems with macro expansion)
+settings.rust.toolchain.expand.macros.label=Expand macros:
+settings.rust.toolchain.expand.macros.new.engine.label=Use new engine
+settings.rust.toolchain.expand.macros.old.engine.label=Use old engine (some features are not supported)
```

Comment:
I'd vote for "some features will not be supported"

Reply:
Why?

Reply:
I've applied your suggestion